source/development/versioning.rst

```diff
@@ -1,4 +1,76 @@
 .. _versioning:
 
-Versioning
-==========
+Relases and Versioning
+======================
+
+Bao Project's repos follow a `semantic versioning scheme <https://semver.org/>`_
+where, for each release, the specific `git commit is tagged
+<https://git-scm.com/book/en/v2/Git-Basics-Tagging>` with  a label with the form
+MAJOR.MINOR.PATCH, where each version component is incremented as following:
+
+* MAJOR version when you make incompatible API changes
+* MINOR version when you add functionality in a backward compatible manner
+* PATCH version when you make backward compatible bug fixes
+
```

Comment:
```suggestion
* **MAJOR** version when you make incompatible API changes
* **MINOR** version when you add functionality in a backward compatible manner
* **PATCH** version when you make backward compatible bug fixes
```

Reply:
Added the bold for the version components.